cob_dashboard/scripts/cob_dashboard_aggregator.py

```diff
@@ -47,46 +48,31 @@ def __init__(self):
     self.pub = rospy.Publisher("dashboard_agg", DashboardState, queue_size=1)
 
     # Create subscribers
-    # Circuit Breaker
-    rospy.Subscriber("power_board/state", PowerBoardState, self.powerBoardCB)
-    self.last_power_board_state = 0
-    # Battery
-    rospy.Subscriber("power_state", PowerState, self.powerCB)
-    self.last_power_state = 0
-    # Wireless
-    rospy.Subscriber("ddwrt/accesspoint", AccessPoint, self.accessPointCB)
-    self.last_access_point = 0
```

Comment:
accesspoint monitoring is not working since years and was never used. I now only use the features which are working and supported on all robots.